jquery.flot.composeImages.js

```diff
@@ -179,5 +179,17 @@
         return GENERALFAILURECALLBACKERROR;
     }
 
+    // used for testing
     $.plot.composeImages = composeImages;
```

Comment:
Do not forget to expose this in jquery.flot.js as public function for plot :)

Reply:
@VirgilChindris already did this instead of me :)
https://github.com/ni-kismet/engineering-flot/blob/master/API.md#utils